packages/altair-app/src/app/modules/altair/config.ts

```diff
@@ -2,6 +2,12 @@ import isElectron from './utils/is_electron';
 import { IDictionary } from './interfaces/shared';
 import { IInitialEnvironments } from './store/environments/environments.reducer';
 import * as fromSettings from './store/settings/settings.reducer';
+import type {
+  ACTION_CABLE_PROVIDER_ID,
+  APP_SYNC_PROVIDER_ID,
+  GRAPHQL_WS_PROVIDER_ID,
+  WEBSOCKET_PROVIDER_ID
+} from "./services/subscriptions/subscription-provider-registry.service";
```

Comment:
Can we import these from a single file such that we could do `import * as subscriptionProviderKeys from '../file'`? I would want to have to reduce the number of files to change when adding new subscription providers in the future.